deeppavlov/models/elmo/elmo.py

```diff
@@ -272,6 +273,7 @@ def __init__(self,
         self._build_model(train=False, epoch=load_epoch_num)
 
         self.save()
+        self.load_path = self.save_path
```

Comment:
Please comment on this or use a different variable for reloading model while training